### PR TITLE
Azure Cognitive Search - TOC fix, landing page corrections

### PR DESCRIPTION
**Pull request by HeidiSteen**

### CODE REVIEW

File: api/overview/azure/search.md [hunk 1]
```diff
@@ -14,21 +14,25 @@ ms.service: search
...
 ## Libraries for data access
 
-The `Azure.Search.Documents` client library (v1) is a new offering for .NET developers who want to use search technology in their applications. This version supersedes the `Microsoft.Azure.Search` client library (v10) with full functional parity.
+After an Azure Cognitive Search resource is created and configured, use data access libraries to create and consume search objects in client applications.
 
-The APIs are similar but not identical. When reviewing code samples and content, be sure to check for the namespace (`using Azure.Search.Documents;`) to confirm which client ibrary is in use.
+### Version 11
 
-### Version 1.x
+The **`Azure.Search.Documents`** is a new client library  for .NET developers who want to use search technology in their applications. This version is functionally equivalent to the v10 legacy client library, but takes dependencies on **`Azure.Core`** and **`System.Text.Json`** to provide a standard approach for service configuration, authentication, document serialization, and other tasks.
```
Comment (brjohnstmsft) [inline api/overview/azure/search.md:21]: The version that's available right now isn't actually "functionally equivalent" yet, but the August release should be. Do we want to delay publishing this a bit, or put a footnote here or something?

CC @tg-msft @heaths for their opinions

File: api/overview/azure/search.md [hunk 2]
```diff
@@ -14,21 +14,25 @@ ms.service: search
...
 
-### Version 10.x (legacy)
+There is only one package and one client library for this version.
 
-Use the following version 10.x of the **Microsoft.Azure.Search** libraries to work with Azure Cognitive Search objects. With the exception of bug fixes, no further updates are planned for this library.
+If you have existing search applications that call the v10 legacy libraries, be aware that this version has different clients, namespaces, and class names. You will need to migrate existing code to use the new library. When reviewing code samples and content, be sure to check for the namespace (`using Azure.Search.Documents;`) to confirm whether the v11 client library is demonstrated.
+
+### Version 10 (legacy)
+
+Use **`Microsoft.Azure.Search`** libraries to maintain existing solutions. This version is fully supported, but with the exception of bug fixes, no further updates are planned for this library. For an introduction, see [How to use Microsoft.Azure.Search (v10)](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).
```
Comment (brjohnstmsft) [inline api/overview/azure/search.md:35]:
```suggestion
Use **`Microsoft.Azure.Search`** libraries to maintain existing solutions. This version is still supported for now, but with the exception of security hotfixes, no further updates are planned for this library. For an introduction, see [How to use Microsoft.Azure.Search (v10)](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).

```

Comment (brjohnstmsft) [reply api/overview/azure/search.md:35]: It's important that we don't overcommit here.

File: api/overview/azure/search.md (hunk 1 above)
Comment (HeidiSteen) [reply api/overview/azure/search.md:21]: Neither v10 nor v11 have knowledge store, but they do have skillsets.  Both target 2019-05-06 REST API.  In that regard, I see them as functionally equivalent.  In August, v11 will be ahead when it adds kstore support (and thus targets 2020-06-30).  I can change the wording then?

File: api/overview/azure/search.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ If you have existing search applications that call the v10 legacy libraries, be
 
 ### Version 10 (legacy)
 
-Use **`Microsoft.Azure.Search`** libraries to maintain existing solutions. This version is fully supported, but with the exception of bug fixes, no further updates are planned for this library. For an introduction, see [How to use Microsoft.Azure.Search (v10)](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).
+Use **`Microsoft.Azure.Search`** libraries to maintain existing solutions. This version is fully supported for now, but with the exception of security hot fixes, no further updates are planned for this library. For an introduction, see [How to use Microsoft.Azure.Search (v10)](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).
```
Comment (brjohnstmsft) [inline api/overview/azure/search.md:35]:
```suggestion
Use **`Microsoft.Azure.Search`** libraries to maintain existing solutions. This version is supported for now, but with the exception of security hotfixes, no further updates are planned for this library. For an introduction, see [How to use Microsoft.Azure.Search (v10)](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).

```

Comment (brjohnstmsft) [reply api/overview/azure/search.md:35]: "Fully" says more than it should; "hotfix" is one word

**Comment by HeidiSteen** [comment]
#sign-off  @scbedd @maggiepint 
Please merge for the Tuesday July 21 publish.  This PR restores a landing page for the older V10 client, adds client version disambiguation in the TOC, and adds specifics about version differences. Only my files were touched in this PR. The validation warning in this PR is unrelated to my changes.

**Pull request merged**